addons/Drop/drop.lua

```diff
@@ -0,0 +1,65 @@
+_addon.name = 'Drop'
+_addon.author = 'Arico'
+_addon.version = '1'
+_addon.commands = {'drop'}
+
+
+packets = require('packets')
+require('luau')
+res = require('resources')
+require('logger')
+
+check_inv = function(item_id)
+        for k, v in pairs(windower.ffxi.get_items().inventory) do

```

Comment:
`get_items` is a *very* expensive function call (the most or second most expensive one in our API, depending on the situation), and you call it both here and in `drop_item`. You should call it before either and pass it in to both. That said, you can actually condense the logic into one function, then you wouldn't need to do it, see below.

Also, if you only want the inventory pass it in as a parameter:
```lua
windower.ffxi.get_items('inventory') -- the name of the bag
-- or
windower.ffxi.get_items(0) -- or the ID
```